### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by floatdrop**
- Moved fallbacks under single folder
- Split OS related code to `lib` folder
- Simplified `handler` in termux

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,82 +1,8 @@
...
-				return execa.sync('xsel', ['--clipboard', '--output'], opts);
-			} catch (err) {
-				handler(err);
-			}
-		}
-	}
-};
+const linux = require('./lib/linux.js');
+const darwin = require('./lib/darwin.js');
+const win32 = require('./lib/win32.js');
```
Comment (sindresorhus) [inline index.js]: I prefer `win32` => `windows` and `darwin` => `macos`. I'm not really concerned about matching the bad posix names. Same for the folder names in `fallbacks`.

**Comment by sindresorhus** [comment]
Great! Was hoping someone would do this.

You need to remove `vendor` from `files` in package.json.

File: index.js (hunk 1 above)
Comment (floatdrop) [inline index.js]: 👍

**Comment by floatdrop** [comment]
@sindresorhus done.

**Pull request merged**